<ui version="4.0" >
 <class>AUTHCODE</class>
 <widget class="QDialog" name="AUTHCODE" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AuthCode</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>90</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="auth_image" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap/>
   </property>
  </widget>
  <widget class="QLineEdit" name="auth_code" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>50</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>请输入验证码</string>
   </property>
  </widget>
  <widget class="QLabel" name="auth_text" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>130</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="textFormat" >
    <enum>Qt::PlainText</enum>
   </property>
   <property name="wordWrap" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="auth_tips" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>180</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="textFormat" >
    <enum>Qt::PlainText</enum>
   </property>
   <property name="wordWrap" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AUTHCODE</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AUTHCODE</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
